app/server.go

```diff
@@ -152,16 +153,29 @@ func (a *App) StartServer() {
 		handler = httpRateLimiter.RateLimit(handler)
 	}
 
-	a.Srv.GracefulServer = &graceful.Server{
```

Comment:
Not really any reason to use this library since Go 1.8 introduced a graceful shutdown method.